Documentation/Building the Extension.md

```diff
@@ -13,3 +13,12 @@ Prerequisite steps:
   * **(optional)** Set an environment variable `CPPTOOLS_DEV=1`.
```

Comment:
Lines 13 14 can be removed due to https://github.com/microsoft/vscode-cpptools/commit/62fff2b37b0ac5d3f04e01192191731db1c732b6#diff-c593d2f539ed1508705a877abb8a2d2b935d53950f8bde51ad05ac84bc6f2b18

Reply:
Thanks for the suggestion! [888efa9](https://github.com/microsoft/vscode-cpptools/pull/8788/commits/888efa96850d9719c7f58e30fb2d3b88cd3d9314) removed lines 13 and 14.